company_linkage/airtable_configs/company_annotation/000_main.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
The silly `000_` naming is just to ensure this one gets run first, as it's used in the groups merge query. I'll implement a better solution eventually...